wiki/BanchoBot/en.md

```diff
@@ -1,154 +1,143 @@
+`!where` shows the entered user's current location. By default, this only shows the user's country. But if the user has `Share your city location with others` enabled, it will also display their city.* An example of sending this command is shown below:
 
 ```
-pippi: !where Ephemeral
-BanchoBot: Ephemeral is in Australia
+13:02 pippi: !where Ephemeral
+13:02 BanchoBot: Ephemeral is in Australia
 ```
 
 ### FAQ
 
 ```
-!FAQ <entry>
-!FAQ list
+!faq <entry>
 ```
 
-Shows the message of the entry. You can use the `list` argument to show all available entries. When a moderator uses this in a public channel, the response will be outputted to that channel. When normal users use this in a public channel, the response will be outputted through BanchoBot's PM.
+```
+!faq list
+```
 
-#### Usage Examples
+`!faq` shows the contents of the entry. Alternatively, the `list` argument can be used to show all available entries. By default, BanchoBot will respond in English, but it is possible to recieve an answer in another language by prefixing the entry with said language's [two-letter code](/wiki/Article_Styling_Criteria#locales). Examples of sending this command are shown below:
 
 ```
-pippi: !faq peppy
-BanchoBot: peppy is the lead developer and indeed, the creator of osu! and handles most of the project himself.
+13:03 pippi: !faq peppy
+13:03 BanchoBot: peppy is the lead developer and indeed, the creator of osu! and handles most of the project himself.
 ```
 
 ```
-Tama: 你好
-Yuzu: !faq chinese
-BanchoBot: 中文用户请点击 #chinese 以进入中文频道进行交流。
-BanchoBot: #osu 是英文专属频道，如果接获此讯息后继续在 #osu 内以中文交谈，管理员有权利禁言。
-Tama: ok
+13:04 <pipppi> faq ru:lines
+14:04 <BanchoBot> Умещайте свои мысли в меньшее количество строк, чтобы не получить сайленс.
 ```
 
-### REPORT
+### Report
 
-*Notice: If you want to report a moderator, send an email to [support@ppy.sh](mailto:support@ppy.sh) for moderator complaints.*
-
-*Note: You can also use the user card to report a user.*
+*For information about what is worth reporting, see [Reporting Bad Behaviour](/wiki/Reporting_Bad_Behaviour).*
 
 ```
-!REPORT <user> <reason>
+!report <user> <reason>
 ```
 
-*For information about what is worth reporting, see [Reporting Bad Behaviour](/wiki/Reporting_Bad_Behaviour).*
-
-*Notice: When entering the username, replace spaces with underscores (`_`).*
-
-Sends a report to the [Global Moderation Team](/wiki/Global_Moderation_Team) or the [Language Moderators](/wiki/Language_Moderators).
-
-#### Usage Example
+`!report` notifies the [Global Moderation Team](/wiki/Global_Moderation_Team) about a user's inappropriate behaviour; it is also possible to report a user by right-clicking their in-game user card. If a user has spaces in their username, replace them with underscores (`_`). E.g., `really cool username` becomes `really_cool_username`. To report a moderator, send an email to [support@ppy.sh](mailto:support@ppy.sh). An example of reporting a normal user through BanchoBot is shown below:
```

Comment:
if you merge two short sentences, the resulting one looks a bit better:
```suggestion
`!report` notifies the [Global Moderation Team](/wiki/Global_Moderation_Team) about a user's inappropriate behaviour; it is also possible to report a user by right-clicking their in-game user card. If a user has spaces in their username, replace them with underscores (`_`), e.g., `really cool username` becomes `really_cool_username`. To report a moderator, send an email to [support@ppy.sh](mailto:support@ppy.sh). An example of reporting a normal user through BanchoBot is shown below:
```

Reply:
I'd have to disagree with you on that one: it sounds a bit mouthy and teeters on the edge of run-on. It could also be mistaken as referring to the clause that came before it rather than the one that comes after.